### DISCUSSION TITLE
解析出节点 loc 和 range 并不能对应上

### DISCUSSION THREAD

**Discussion by minikey** [wxmlfile/wxml-parser#33] [Q&A]
如果使用 eslint 的 sourceCode 对象不能正确的读取内容
比如：
```
const sourceCode = context.getSourceCode();
console.log(sourceCode.getText(node));
```

<img width="435" alt="image" src="https://user-images.githubusercontent.com/8241628/193033522-5b2111ff-e226-4197-aa92-c012b3c183b6.png">

**Comment by iChenLei** [comment]
偏移1个位置？

**Reply by minikey** [reply]
对的，偏移一个位置
